index.html

```diff
@@ -7631,6 +7631,11 @@ <h3><dfn data-lt="adding a cookie">Add Cookie</dfn></h3>
    <dd><p>The value if the entry exists, otherwise leave unset to
     indicate that this is a session cookie.
 
+    <aside class=note>
+      <p>The cookie&apos;s expiry value might be limited by the remote end in
+       accordance with the <a>Cookie Lifetime Limits</a>.
```

Comment:
@jgraham I replaced `as required` by `in accordance` because `required` is not allowed per Bikeshed and would cause a failure when building the BiDi specification, and I think that we should have the same wording in both specs.